tests/topotests/lib/topogen.py

```diff
@@ -118,6 +118,25 @@ def exacmd_version_ok(exacmd):
     return None
 
 
+def check_pmacct_version(min_version="1.7.6", commander=None):
+    """Check for the pmacct version"""
+
+    if commander is None:
+        commander = Commander("topogen")
+
+    _, stdout, _ = commander.cmd_status("pmbmpd -V", warn=False)
```

Comment:
Is this installed on the CI?

Reply:
This daemon comes with the **"pmacct"** package
In ubuntu **20.04** we have the version **1.7.2** and it is very poor in term of bmp features, the present test is modified to support a very basic cases so it should be removed when a newer version is available.

In ubuntu **22.04** we have the version **1.7.6**, the tests should pass fine.

I don't know if FRR CI will support BMP tests using this package as a BMP collector (no good replacement has been found!). If not so, the need for an intern FRR BMP collector should be implemented.